parts/player/player.lua

```diff
@@ -1235,15 +1268,15 @@ function Player:resetBlock()-- Reset Block's position and execute I*S
 
     if not ENV.logicalIRS then
         -- If logical IRS is disabled, all IRS inputs will be buffered to prevent survival.
-        self.bufferedIRS=true
-        self.bufferedDelay=0
-        if ENV.wait==0 then
-            self.bufferedDelay=ENV.irscut
+        self.bufferedIRS = true
+        self.bufferedDelay = 0
+        if ENV.wait == 0 then
+            self.bufferedDelay = ENV.irscut
         end
-    elseif ENV.wait==0 and ENV.irscut>0 and not self:ifoverlap(C.bk,self.curX,self.curY) then
+    elseif ENV.wait==0 and ENV.irscut>0 and not self:ifoverlap(C.bk, self.curX, self.curY) then
         -- If IRS cut delay is enabled and we aren't currently dying, buffer the input instead.
-        self.bufferedIRS=true
-        self.bufferedDelay=ENV.irscut
+        self.bufferedIRS = true
+        self.bufferedDelay = ENV.irscut
```

Comment:
Electra, MrZ has a weird code style
He used to make a game also named "Techmino" on a graohic calculator, since it is memory limited, this is why he never leave a space on where he thinks "unnecessary" or "memory wasted"